projects/tera/api/src/Service/PlantVerifier.php

```
@@ -0,0 +1,27 @@
+
+readonly class PlantVerifier
+{
+    public function __construct(private HttpClientInterface $floraClient)
+    {
+
+    }
+
+    public function assertPlantExists(string $plantUlid): void
+    {
+        try {
+            $response = $this->floraClient->request(
```

PHPMD (2.15.0)

Avoid unused local variables such as '$response'. (Unused Code Rules)

(UnusedLocalVariable)